### DISCUSSION TITLE
Community Discord server

### DISCUSSION THREAD

**Discussion by henryglends** [sanni/cartreader#555] [General Chat]
Hey, so I know this is kind of a weird question, but what are the chances of a community discord server? I feel like this could help bring in more retro gamers, and more people who just want to revive their games or hang out somewhere and just learn.

**Comment by zombie343** [comment]
This would be great!

**Comment by Ancyker** [comment]
Well, since no one else did it I went ahead and created one...

https://discord.gg/YVxv7BnzXt

**Comment by nsx0r** [comment]
this is a good idea
